mysys_ssl/CMakeLists.txt

```diff
@@ -39,6 +40,7 @@ SET(MYSYS_SSL_SOURCES
     my_md5.cc
     my_rnd.cc
     my_murmur3.cc
+    my_crypt.cc
```

Comment:
Does incremental YaSSL configuration rebuild work, if only yassl.cc is touched?

Reply:
Yes, it works. Change to yassl.cc equals change to my_crypt.cc as it includes whole yassl.cc file.